assets/js/script.js

```diff
@@ -489,25 +496,65 @@ function defineWord(word, id) {
 
 function getBoxValues(category, num) {
 	var box = {};
-	for (let value of Object.values(category)) {
-		if (typeof box === 'object' && box !== null) {
+	for (i = 0; i < 5; i++) {
+		if (typeof box === 'object' && box) {
 			// console.log('box 1 statement 1 true');
-			if (value.value == 100 * num) {
+			if (num === 1) {
+				// console.log('100 value here');
+				box.value = 100;
+				box.question = category[num - 1].question;
+				box.answer = category[num - 1].answer;
+			}
+			if (num === 2) {
+				// console.log('100 value here');
+				box.value = 200;
+				box.question = category[num - 1].question;
+				box.answer = category[num - 1].answer;
+			}
+			if (num === 3) {
 				// console.log('100 value here');
-				box.value = value.value;
-				box.question = value.question;
-				box.answer = value.answer;
-			} else if (value.value == 200 * num) {
-				// console.log('200 value here');
-				box.value = value.value;
-				box.question = value.question;
-				box.answer = value.answer;
+				box.value = 300;
+				box.question = category[num - 1].question;
+				box.answer = category[num - 1].answer;
+			}
+			if (num === 4) {
+				// console.log('100 value here');
+				box.value = 400;
+				box.question = category[num - 1].question;
+				box.answer = category[num - 1].answer;
+			}
+			if (num === 5) {
+				// console.log('100 value here');
+				box.value = 500;
+				box.question = category[num - 1].question;
+				box.answer = category[num - 1].answer;
 			}
 		}
 	}
 	return box;
 }
 
+// function getBoxValues(category, num) {
```

Comment:
This looks like a duplicate function?  seeing the same one on line 490

Reply:
This was what was there before. I refactored but left the old one in, just commented out. In case we need it